nuxeoenv.sh

```diff
@@ -296,6 +296,5 @@ get_opts "$@"
 get_input
 parse_input
 venv_init
-venv_activate
```

Comment:
venv_init is doing activation yes but only when creating the ./venv/, 
for successive invocation, venv_activate is needed